src/elements/basic/ButtonElement.jsx

```diff
@@ -49,6 +49,7 @@ function applyButtonStyles(element, applyStyles) {
       padding: `${a}px ${b}px ${c}px ${d}px`
     })
   );
+  applyStyles.apply('tc', 'text_align', (a) => ({ justifyContent: a }));
```

Comment:
where do you add the tc target?

Reply:
ah, ty, good catch! I forgot to add it for img as well, both are now added